src/site/content/en/blog/launch-handler/index.md

```diff
@@ -0,0 +1,244 @@
+  where it generally makes sense to only have one instance of the app open at any time.
+- Apps that enable multi-document management, but within their own single instance, for example, an
+  HTML-implemented tab strip, floating sub-windows, or apps using
+  [tabbed application mode](/tabbed-application-mode/).
+
+## Current status {: #status }
+
+<div class="w-table-wrapper">
+
+| Step                                     | Status                   |
+| ---------------------------------------- | ------------------------ |
+| 1. Create explainer                      | [Complete][explainer]    |
+| 2. Create initial draft of specification | Not started              |
+| 3. Gather feedback & iterate on design   | [In progress](#feedback) |
+| 4. **Origin trial**                      | [**Started**][ot]        |
+| 5. Launch                                | Not started              |
+
+</div>
+
+## How to use the Launch Handler API {: #use }
+
+### Enabling via about://flags
+
+To experiment with the Launch Handler API locally, without an origin trial token, enable the
+`#enable-desktop-pwas-launch-handler` flag in `about://flags`.
+
+### Enabling support during the origin trial phase
+
+Starting in Chromium&nbsp;98, the Launch Handler API will be available as an origin trial in
+Chromium. The origin trial is expected to end in Chromium&nbsp;102 (June&nbsp;15, 2022).
+
+{% include 'content/origin-trials.njk' %}
+
+### Register for the origin trial {: #register-for-ot }
+
+{% include 'content/origin-trial-register.njk' %}
+
+### Feature detection
+
+To check if the Launch Handler API is supported, use:
+
+```javascript
+if ('launchQueue' in window && 'targetURL' in LaunchParams.prototype) {
+  // The Launch Handler API is supported.
+}
+```
+
+### The `launch_handler` manifest member
+
+To declaratively specify the launch behavior of your app, add the `launch_handler` manifest member
+to your manifest. It has two sub-fields, `route_to` and `navigate_existing_client`. The former lets
+you control whether a new or an existing client should be launched, and the latter how and if this
+client should be navigated. The code snippet below shows an excerpt of a Web App Manifest file with
+exemplary values that would always route all launches to a new client.
+
+```json
+{
+  "launch_handler": {
+    "route_to": "new-client",
+    "navigate_existing_client": "always"
+  }
+}
+```
+
+If unspecified, `launch_handler` defaults to
+`{"route_to": "auto", "navigate_existing_client": "always"}`. The allowed values for the sub-fields
+are as follows:
+
+- `route_to`:
+
+  - `new-client`: A new browsing context is created in a web app window to load the launch's target
+    URL.
+  - `existing-client`: The most recently interacted with browsing context in a web app window for
+    the app being launched is chosen to handle the launch. How the launch is handled within that
```

Comment:
```suggestion
  - `existing-client`: An existing browsing context is used, specifically the one most recently used. How the launch is handled within that
```

Reply:
The exiting sentence was a bit of a mouthful. If I didn't get this right, we'll need a third option.